doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -445,6 +472,12 @@ Libraries for networking
   * Updated:
 
     * The stack size of the MQTT connection monitoring thread can now be adjusted by setting the :kconfig:option:`CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD_STACK_SIZE` Kconfig option.
+    * The library now subscribes to a wildcard cloud-to-device (c2d) topic.
```

Comment:
After some discussion, we decided the abbreviations for C2D, D2C, and so on should be with capital letters.

Reply:
the topic string is `"c2d"`, so I would prefer to keep it lower case.
how should i format it? just add quotes?

Reply:
No quotes or other formatting, just C2D

If you want to keep it lowercase, use ``/c2d`` to refer to the exact topic text.